daml-lf/data-scalacheck/src/main/scala/com/digitalasset/daml/lf/data/DataArbitrary.scala

```diff
@@ -6,16 +6,17 @@ package com.digitalasset.daml.lf.data
 import ImmArray.ImmArraySeq
 import org.scalacheck.{Arbitrary, Gen}
 import Arbitrary.arbitrary
+import scalaz.{@@, Tag}
 
 object DataArbitrary {
-  implicit def arbFrontStack[A: Arbitrary]: Arbitrary[FrontStack[A]] =
+  implicit def `arb FrontStack`[A: Arbitrary]: Arbitrary[FrontStack[A]] =
```

Comment:
Just so I understand, you use this convention of adding spaces to names as a way to mark members to not be used explicitly, right?

Reply:
The main goal is to avoid name collisions; what you state is a secondary, albeit also useful goal.